packages/language-server/src/plugins/collector/collectSemanticToken.ts

```diff
@@ -0,0 +1,18 @@
+import { Range, SemanticTokens } from 'vscode-languageserver';
```

Comment:
Can we inline this function into `PluginHost`? While I appreciate moving code into separate files, I think in this instance it breaks the nice "all folders within the plugins folder are a plugin"-semantic. If I understand your other code comment we might even remove this altogether since only the TS plugin provides semantic tokens.